### DISCUSSION TITLE
Improving reviewing pull requests

### DISCUSSION THREAD

**Discussion by tmg-magna** [community/community#110969] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

For pull requests, in the "files changed" tab, it would be great if the "Viewed" checkbox would be at the bottom of the file. It does take unnecessary effort to scroll back up, especially for large files, just to tick that box. It would really improve the review process if that check box was at the bottom so that one can view the file, tick it as viewed and then move on to the next file.

I have attached a screenshot of the element I am referring too.

![image](https://github.com/community/community/assets/146425336/5877e33d-7e21-47e7-b298-55759e9eb581)

**Comment by torbjokv** [comment]
looks like duplicate of https://github.com/orgs/community/discussions/15149 and https://github.com/orgs/community/discussions/9676
